.github/workflows/hq-smoke-tests.yml

```
@@ -108,10 +108,16 @@ jobs:
                 .replace(/{{runNumber}}/g, context.runNumber)
                 .replace(/{{environment}}/g, CC_ENV)
 
+            let receivers = 'qa-automation@dimagi.com'
+            if (context.eventName !== "pull_request") {
```

It would still trigger on push,  right? Maybe we add a check for that as well -- not sure if the syntax is correct, but something like:

 ```suggestion
            if (context.eventName !== "pull_request" || context.eventName !== "push" ) {
```

---

https://github.com/dimagi/dimagi-qa/commit/c3a39cd0d2593186eea919592b0a78f8dd363157